vulnerabilities/tests/test_redhat_importer.py

```diff
@@ -83,18 +85,38 @@ def test_to_advisory(self):
                         Reference(
                             url="https://bugzilla.redhat.com/show_bug.cgi?id=1396383",
                             reference_id="1396383",
+                            severities=[
+                                VulnerabilitySeverity(
+                                   system=scoring_systems["rhbs"], value=2.0
+                                )
+                            ],
                         ),
                         Reference(
                             url="https://access.redhat.com/errata/RHSA-2017:1931",
                             reference_id="RHSA-2017:1931",
+                            severities=[
+                                VulnerabilitySeverity(
+                                    system=scoring_systems["rhas"], value=2.2
+                                )
+                            ],
                         ),
                         Reference(
                             url="https://access.redhat.com/errata/RHSA-2017:0725",
                             reference_id="RHSA-2017:0725",
+                            severities=[
+                                VulnerabilitySeverity(
+                                    system=scoring_systems["rhas"], value=2.2
+                                )
+                            ],
                         ),
                         Reference(
                             url="https://access.redhat.com/hydra/rest/securitydata/cve/CVE-2016-9401.json",  # nopep8
                             reference_id="",
+                            severities=[
+                                VulnerabilitySeverity(
+                                    system=scoring_systems["rh_cvssv3"], value=6.0
```

Comment:
this is not a scoring system

Reply:
A CVSSV3 VulnerabilitySeverity created by RedHat is already in the context of a RHSA Reference above. RedHat just uses the same principles as the NVD when the use a CVSSv3, e.g. the same system. They just provide different values.